README.md

```diff
@@ -54,182 +47,191 @@ All URIs are relative to *https://api.factern.com/v2*
-*FacternApi.FactsApi* | [**describe**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#describe) | **POST** /describe | Describe
-*FacternApi.FactsApi* | [**history**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#history) | **POST** /history | History
-*FacternApi.FactsApi* | [**label**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#label) | **POST** /label | Label a Node
-*FacternApi.FactsApi* | [**obliterate**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#obliterate) | **POST** /obliterate | Obliterating Information Nodes
-*FacternApi.FactsApi* | [**permission**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#permission) | **POST** /permission | Create Permission
-*FacternApi.FactsApi* | [**read**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#read) | **POST** /read | Reading
-*FacternApi.FactsApi* | [**replaceInformation**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#replaceInformation) | **POST** /replaceinformation | Replace
-*FacternApi.FactsApi* | [**requestPermission**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#requestPermission) | **POST** /requestpermission | Request Permission
-*FacternApi.FactsApi* | [**resetLogin**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#resetLogin) | **POST** /resetlogin | Changing Login Password
-*FacternApi.FactsApi* | [**searchAlias**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#searchAlias) | **POST** /searchalias | Search For Alias
-*FacternApi.FactsApi* | [**searchEntity**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#searchEntity) | **POST** /searchentity | Search For Entity
-*FacternApi.FactsApi* | [**settleAccount**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#settleAccount) | **POST** /settleaccount | Settle Account
-*FacternApi.FactsApi* | [**updateApplication**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#updateApplication) | **POST** /updateapplication | Resetting Application Secret
-*FacternApi.FactsApi* | [**updateStatus**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#updateStatus) | **POST** /updatestatus | Enabling/Disabling Nodes
-*FacternApi.FactsApi* | [**watch**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#watch) | **POST** /watch | Create Watch Trigger
-*FacternApi.FactsApi* | [**write**](https://github.com/Factern/factern-client-javascript/blob/master/docs/FactsApi.md#write) | **POST** /write | Writing by Template
+*FacternApi.FactsApi* | [**addMember**](docs/FactsApi.md#addMember) | **POST** /createmember | Create Member
```

Comment:
I don't believe these relative links will work on npm, I had to do a search/replace before to link to github